gtk/gtk.go

```diff
@@ -9506,6 +9522,130 @@ func (v *TreeRowReference) Valid() bool {
 	return gobool(c)
 }
 
+/*
+ * GtkTreeModelSort
+ */
+
+// TreeModelSort is a representation of GTK's GtkTreeModelSort
+type TreeModelSort struct {
+	*glib.Object
+
+	// Interfaces
+	TreeModel
+}
+
+// native returns a pointer to the underlying GtkTreeModelSort
+func (v *TreeModelSort) native() *C.GtkTreeModelSort {
+	if v == nil || v.GObject == nil {
+		return nil
+	}
+	p := unsafe.Pointer(v.GObject)
+	return C.toGtkTreeModelSort(p)
+}
+
+func marshalTreeModelSort(p uintptr) (interface{}, error) {
+	c := C.g_value_get_object((*C.GValue)(unsafe.Pointer(p)))
+	obj := glib.Take(unsafe.Pointer(c))
+	return wrapTreeModelSort(obj), nil
+}
+
+func wrapTreeModelSort(obj *glib.Object) *TreeModelSort {
+	tm := wrapTreeModel(obj)
+	return &TreeModelSort{obj, *tm}
+}
+
+func (v *TreeModelSort) toTreeModel() *C.GtkTreeModel {
+	if v == nil {
+		return nil
+	}
+	return C.toGtkTreeModel(unsafe.Pointer(v.GObject))
+}
+
+func (v *TreeModelSort) toTreeSortable() *C.GtkTreeSortable {
+	if v == nil {
+		return nil
+	}
+	return C.toGtkTreeSortable(unsafe.Pointer(v.GObject))
+}
+
+// TreeModelSortNew is a wrapper around gtk_tree_model_sort_new_with_model():
+func TreeModelSortNew(model ITreeModel) (*TreeModelSort, error) {
+	var mptr *C.GtkTreeModel
+	if model != nil {
+		mptr = model.toTreeModel()
+	}
+	c := C.gtk_tree_model_sort_new_with_model(mptr)
+	if c == nil {
+		return nil, nilPtrErr
+	}
+
+	tms := wrapTreeModelSort(glib.Take(unsafe.Pointer(c)))
+	return tms, nil
+}
+
+// GetModel is a wrapper around gtk_tree_model_sort_get_model()
+func (v *TreeModelSort) GetModel() ITreeModel {
+	c := C.gtk_tree_model_sort_get_model(v.native())
+	if c == nil {
+		return nil
+	}
+	m := wrapTreeModel(glib.Take(unsafe.Pointer(c)))
+	return m
+}
+
+// ConvertChildPathToPath is a wrapper around gtk_tree_model_sort_convert_child_path_to_path().
+func (v *TreeModelSort) ConvertChildPathToPath(childPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_sort_convert_child_path_to_path(v.native(), childPath.native())
+	if path == nil {
+		return nil
+	}
+	p := &TreePath{path}
+	return p
+}
+
+// ConvertChildIterToIter is a wrapper around gtk_tree_model_sort_convert_child_iter_to_iter().
+func (v *TreeModelSort) ConvertChildIterToIter(sortIter, childIter *TreeIter) bool {
+	return gobool(C.gtk_tree_model_sort_convert_child_iter_to_iter(v.native(), sortIter.native(), childIter.native()))
+}
+
+// ConvertPathToChildPath is a wrapper around gtk_tree_model_sort_convert_path_to_child_path().
+func (v *TreeModelSort) ConvertPathToChildPath(sortPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_sort_convert_path_to_child_path(v.native(), sortPath.native())
+	p := &TreePath{path}
+	return p
+}
```

Comment:
the return value of gtk_tree_model_sort_convert_path_to_child_path() may be NULL

please see `func (v *TreeModelFilter) ConvertPathToChildPath(filterPath *TreePath)` as reference for fixing